src/tiles/clickable-tile.component.ts

```diff
@@ -0,0 +1,70 @@
+import {
+	Component,
+	Input
+} from "@angular/core";
+
+/**
+ * Build application's clickable tiles using this component.
+ *
+ * ## Basic usage
+ *
+ * ```html
+ * <ibm-clickable-tile>
+ * 		tile content
+ * </ibm-clickable-tile>
+ * ```
+ *
+ * @export
+ * @class ClickableTile
+ * @implements {OnInit}
+ */
+@Component({
+	selector: "ibm-clickable-tile",
+	template: `
+	<a
+		[ngClass]="{
+			'bx--link': true,
+			'bx--tile': true,
+			'bx--tile--clickable': true,
+			'bx--tile--is-clicked': clicked
+		}"
+		tabindex="0"
+		(click)="onClick($event)"
+		(keydown)="onKeyDown($event)"
+		[href]="href"
+		[attr.aria-disabled]="disabled">
+		<ng-content></ng-content>
+	</a>`
+})
+export class ClickableTile {
+	static CARRIAGE_RETURN = "Enter";
+	static SPACE = " ";
+
+	/**
+	 * Sets the `href` attribute on the `ibm-clickable-tile` element.
+	 * @type {string}
+	 * @memberof ClickableTile
+	 */
+	@Input() href = "";
+
+	/**
+	 * Sets the `aria-disabled` attribute on the `ibm-clickable-tile` element.
```

Comment:
This description is not really needed but if you wanna keep it, put it after "Set to `true` to disable the clickable tile."
That is what people really need to know about this input when they read the docs.